canonical-websites/insights.ubuntu.com
# Build new featured posts module

## Done

Build new featured posts section

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- Check that the featured posts section matches the proposed design


## Issue / Card

Fixes #313
